earth_enterprise/src/common/khxml/khXML.cpp

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Minor: I'd recommend changing the names of these files so that we don't have to differentiate between khxml and khXML.

Reply:
Well, ideally it would replace khxml.h/.cpp. if needed, I can change the file name now though.

Reply:
No need to change the name now. Just something to take care of before it's checked in.